client/dashboard/section.js

```diff
@@ -0,0 +1,58 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+export default class StorePerformance extends Component {
+	constructor( props ) {
+		super( props );
+		const { title } = props;
+
+		this.state = {
+			titleInput: title || __( 'Store Performance', 'woocommerce-admin' ),
+		};
+
+		this.onMenuToggle = this.onMenuToggle.bind( this );
+		this.onTitleChange = this.onTitleChange.bind( this );
+		this.onTitleBlur = this.onTitleBlur.bind( this );
+	}
+
+	onMenuToggle( isOpen ) {
```

Comment:
Can we remove this function in favour of placing the logic in `onTitleBlur`? Just a thought here, I haven't tested the assumption that `onTitleBlur` will definitely get called when closing the dropdown.